drivers/dac/ad5758/ad5758.c

```diff
@@ -285,23 +289,20 @@ int32_t ad5758_set_dc_dc_conv_mode(struct ad5758_dev *dev,
 	 * The ENABLE_PPC_BUFFERS bit must be set prior to enabling PPC current
 	 * mode.
 	 */
-	if (mode == PPC_CURRENT_MODE) {
+	if (dev->dc_dc_mode == PPC_CURRENT_MODE) {
```

Comment:
Is this change intended? Before the if clause here checked the new mode parameter. Now it checks the mode currently saved in the handler.